src/main/resources/application.properties

```diff
@@ -31,6 +31,8 @@ nc.app.url=${nc.app.url.protocol}://${nc.app.url.host}:${nc.app.url.port}${nc.ap
 
 nc.jwt.enabled=false
 nc.processing-info-generator.enabled=true
+nc.taskExecutor.corePoolSize=8
```

Comment:
we use dash notation in properties in notiflow. Means:
nc.task-executor.core-pool-size

And as this is kind of globall setting, maybe we should add that to NcAppConfigProperties